<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora IPV4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>CIDR</string>
    </property>
    <widget class="QRadioButton" name="radioButtonCIDR_Nao">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Não</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonCIDR_Sim">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Sim</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Marco Aurélio Sirqueira de Sá || Danillo Ilggner da Silva Maciel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Máscara (usando ponto &lt;0.0.0.0&gt;)</string>
    </property>
    <widget class="QLineEdit" name="lineEditMascara">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Endereço (usando ponto &lt;0.0.0.0&gt;)</string>
    </property>
    <widget class="QLineEdit" name="lineEditEndereco">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxCIDR">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>42</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>8</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonCalcular">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Sub-redes a serem exibidas (limite máx. 100)</string>
    </property>
    <widget class="QSpinBox" name="spinBoxNumSubRedes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditCIDR">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
